docs/source/contributing/developing.adoc

```diff
@@ -225,23 +225,42 @@ To run all the test, use the following command:
 $ make integration_all
 ----
 
-Flag *_TIMEOUT_* can be used to override the default timeout `3600s`.
-To run all the test with timeout `7200s`, use the following command:
+===== Additional Parameters
+
+To provide more flexibility the default targets `integration` and `integration_all` can be further customized using several parameters.
+
+EXEC_BEFORE_FEATURE::
+Parameter `EXEC_BEFORE_FEATURE` specifies Minishift commands to be run before each feature.
+This provides ability to run integration tests against Minishift which is not in default state.
+When multiple commands are specified, they must be delimited by a semicolon.
+For example, tests can be run against stopped Minishift with _image caching_ option turned on by running:
 
 ----
-$ make integration_all TIMEOUT=7200s
+$ make integration_all EXEC_BEFORE_FEATURE="start; stop; config set image-caching true"
 ----
 
-To run integration tests against a Minishift binary in a different location you can use the `MINISHIFT_BINARY` argument:
+GODOG_OPTS::
+This parameter is covered in chapter <<godog-options>>.
```

Comment:
I do not think we should add this section if it does not have any content. We rename the the godog-options section to get better user attention.